plutus-core/plutus-core/test/Pretty/Golden/Readable/StdLib/Data/List/FoldrList.plc.golden

```diff
@@ -1 +1 @@
-/\(a :: *) -> /\(r :: *) -> \(f : a -> r -> r) -> \(z : r) -> (/\(a :: *) -> /\(b :: *) -> \(f : (a -> b) -> a -> b) -> (/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} (iwrap (\(self :: * -> *) -> \(a :: *) -> self a -> a) (a -> b) (\(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) (a -> b)) -> \(x : a) -> f ((/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} s) x))) {(\(a :: *) -> ifix (\(list :: * -> *) -> \(a :: *) -> all (r :: *). r -> (a -> list a -> r) -> r) a) a} {r} (\(rec : (\(a :: *) -> ifix (\(list :: * -> *) -> \(a :: *) -> all (r :: *). r -> (a -> list a -> r) -> r) a) a -> r) -> \(xs : (\(a :: *) -> ifix (\(list :: * -> *) -> \(a :: *) -> all (r :: *). r -> (a -> list a -> r) -> r) a) a) -> unwrap xs {r} z (\(x : a) -> \(xs' : (\(a :: *) -> ifix (\(list :: * -> *) -> \(a :: *) -> all (r :: *). r -> (a -> list a -> r) -> r) a) a) -> f x (rec xs')))
\ No newline at end of file
+/\(a :: *) -> /\(r :: *) -> \(f : a -> r -> r) -> \(z : r) -> (/\(a :: *) -> /\(b :: *) -> \(f : (a -> b) -> a -> b) -> (/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} (iwrap (\(self :: * -> *) -> \(a :: *) -> self a -> a) (a -> b) (\(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) (a -> b)) -> \(x : a) -> f ((/\(a :: *) -> \(s : (\(a :: *) -> ifix (\(self :: * -> *) -> \(a :: *) -> self a -> a) a) a) -> unwrap s s) {a -> b} s) x))) {list a} {r} (\(rec : list a -> r) -> \(xs : list a) -> (/\(a :: *) -> \(x : list a) -> /\(r :: *) -> \(z : r) -> \(f : a -> list a -> r) -> (/\(a :: *) -> \(b : bool) -> \(x : unit -> a) -> \(y : unit -> a) -> ifThenElse {unit -> a} b x y ()) {r} (null {a} x) (\(u : unit) -> z) (\(u : unit) -> f (head {a} x) (tail {a} x))) {a} xs {r} z (\(x : a) -> \(xs' : list a) -> f x (rec xs')))
```

Comment:
It's funny how `foldr` defined over built-in lists is only a tiny bit smaller than `foldr` defined over Scott-encoded lists.